msg/LandingTargetPose.msg

```diff
@@ -24,3 +24,13 @@ bool abs_pos_valid		# Flag showing whether absolute position is valid
 float32 x_abs			# X/north position of target, relative to origin (navigation frame) [meters]
 float32 y_abs			# Y/east position of target, relative to origin (navigation frame) [meters]
 float32 z_abs			# Z/down position of target, relative to origin (navigation frame) [meters]
+
+bool abs_vel_valid		# Flag showing whether absolute velocity is valid
+float32 vx_abs			# X/north velocity of target, relative to origin (navigation frame) [meters/second]
+float32 vy_abs			# Y/east velocity of target, relative to origin (navigation frame) [meters/second]
+
+bool orientation_valid		# Flag showing whether orientation is valid
+float32[4] q_orientation	# Quaternion representation of orientation w.r.t. local NED frame.
+				# (w, x, y, z) format, zero rotation is (1, 0, 0, 0)
+
+# TOPICS landing_target_pose moving_platform_pose
```

Comment:
There is a [is_static](https://github.com/PX4/PX4-Autopilot/pull/24457/files#diff-1a356aa7922c342bc9fea340b3b616a3695f223d3abe8bb65035b3eef5d3e3e1R5) field in this message. Why do you need a `moving_platform_pose` topic? It's also never used.